lambda_local/main.py

```diff
@@ -124,7 +124,7 @@ def execute(func, event, context):
         err = sys.exc_info()
         result = json.dumps({
             "errorMessage": str(err[1]),
-            "stackTrace": traceback.extract_tb(err[2]),
+            "stackTrace": traceback.format_exc(),
```

Comment:
How about using `traceback.format_tb(err[2])` instead?

Reply:
Good idea, that makes it look more like the error response provided by AWS now.